### PR TITLE
Updated ISSUE_TEMPLATE.md

### LINKED ISSUE #298: Issue template needs update
@singhpratyush @aviaryan sir should there be another point that in issue template that if the assignee fails to send a PR within 7 days after assignment, they will be unassigned from the issue, so that we don't have issues blocked. Also the number of issues at a time we can work on should also be mentioned.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #298 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [ ] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [ ] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [ ] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -4,8 +4,10 @@
 - [ ] Issue Title is in form `Algo/DS Name [Language]`.
 - [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
 - [ ] This Algo/DS is worth adding to this repository.
+- [ ] I ensure that I'm not already working on 2 or more issues.
```
Comment (aviaryan) [inline ISSUE_TEMPLATE.md]: Make it `assigned to 2 or more ..`

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -4,8 +4,10 @@
...
 - [ ] Assign this issue to me (I want to work on this).
 - [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
+- [ ] Unassign me if I fail to send Pull Request a week.
```
Comment (aviaryan) [inline ISSUE_TEMPLATE.md]: Make it like `I understand that I will be unassigned from this issue if I fail...`

**Comment by aviaryan** [comment]
I did some corrections in the last commit. @aashutoshrathi 
Maybe we (@singhpratyush) should try to do minor 'admin' level tasks ourselves, to minimize patch implementation time.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -4,8 +4,10 @@
 - [ ] Issue Title is in form `Algo/DS Name [Language]`.
 - [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
 - [ ] This Algo/DS is worth adding to this repository.
+- [ ] I ensure that I'm not assigned to 2 or more issues.
```
Comment (Monal5031) [inline ISSUE_TEMPLATE.md]: Shouldn't this be `I ensure that I'm not assigned to more than 2 issues.` as we allow 2 issues at a time?

**Pull request merged**